frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -74,28 +74,23 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
             <Tag style={{ cursor: "default", marginLeft: 8 }} color={isRGB && "#1890ff"}>
               {isRGB ? "24-bit" : "8-bit"} Layer
             </Tag>
-            {alpha > 0 && isVisible ? (
-              <Tooltip title="Make this color layer invisible.">
-                <Icon
-                  type="eye"
-                  onClick={() => this.props.onChangeLayer(layerName, "isVisible", false)}
-                  style={{ marginTop: 4, marginLeft: 8, cursor: "pointer" }}
-                />
-              </Tooltip>
-            ) : (
-              <Tooltip title="Make this color layer visible.">
-                <Icon
-                  type="eye-o"
-                  onClick={() => this.props.onChangeLayer(layerName, "isVisible", true)}
-                  style={{ marginTop: 4, marginLeft: 8, cursor: "pointer" }}
-                />
-              </Tooltip>
-            )}
+            {/* TODO adjust types of the icons when upgrading antd. */}
+            <Tooltip title={isDisabled ? "Enable" : "Disable"}>
+              <Icon
```

Comment:
What do you think about using a small switch here instead?
`<Switch size="small" ... />
I'd say this would make it more clear to the user what the current state is. If we decide to use the switch, the comment about adjusting icon types can be dropped.